### PR TITLE
SWAP-1345 Negative values in time allocation should not be allowed

### PR DESCRIPTION
**Pull request by peter-asztalos**
## Description

This PR updates the validation rules for proposal administration, instrument scientist review, and SEP time allocation to disallow negative values (or too large values).

## How Has This Been Tested

- [x] e2e tests
- [x] manual tests

## Fixes

https://jira.esss.lu.se/browse/SWAP-1345

## Changes

- validation rules updated

## Depends on

https://github.com/UserOfficeProject/user-office-lib/pull/28
https://github.com/UserOfficeProject/user-office-backend/pull/167

## Tests included/Docs Updated?

- [x] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: src/components/proposal/ProposalAdmin.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import { administrationProposalValidationSchema } from '@esss-swap/duo-validation/lib/Proposal';
+// import { administrationProposalValidationSchema } from '@esss-swap/duo-validation/lib/Proposal';
```
Comment (jekabs-karklins) [inline src/components/proposal/ProposalAdmin.tsx:1]: I think we should not leave commented code, as SVC keeps the history

File: src/components/review/ProposalTechnicalReview.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import { proposalTechnicalReviewValidationSchema } from '@esss-swap/duo-validation/lib/Review';
+// import { proposalTechnicalReviewValidationSchema } from '@esss-swap/duo-validation/lib/Review';
```
Comment (jekabs-karklins) [inline src/components/review/ProposalTechnicalReview.tsx:1]: And here as well

**Review by jekabs-karklins — APPROVED** [review]
Looks great

File: src/components/proposal/ProposalAdmin.tsx (hunk 1 above)
Comment (peter-asztalos) [reply src/components/proposal/ProposalAdmin.tsx:1]: After the validation package is upgraded I'll restore them.

**Pull request merged**